src/polyline.js

```diff
@@ -86,4 +93,12 @@ polyline.encode = function(coordinates, precision) {
     return output;
 };
 
+polyline.flip = function(input) {
```

Comment:
What do you think about naming this function `fromGeoJSON` and make it return an encoded polyline?